Musician Module/MusicianModule.cs

```diff
@@ -0,0 +1,512 @@
+        {
+            if (StopButton != null)
+            {
+                StopButton.Dispose();
+                StopButton = null;
+            }
+            if (MusicPlayer != null)
+            {
+                MusicPlayer.Dispose();
+                MusicPlayer = null;
+            }
+        }
+
+        /*######################################
+          # PANEL RELATED STUFF BELOW.
+          ######################################*/
+        private Panel BuildHomePanel(WindowBase wndw)
+        {
+            var hPanel = new Panel()
+            {
+                CanScroll = false,
+                Size = wndw.ContentRegion.Size
+            };
+
+            var contentPanel = new Panel()
+            {
+                Location = new Point(hPanel.Width - 630, 50),
+                Size = new Point(630, hPanel.Size.Y - 50 - Panel.BOTTOM_MARGIN),
+                Parent = hPanel,
+                CanScroll = true
+            };
+            var menuSection = new Panel
+            {
+                ShowBorder = true,
+                //Title = "Musician Panel",
+                Size = new Point(hPanel.Width - contentPanel.Width - 10, contentPanel.Height + Panel.BOTTOM_MARGIN),
+                Location = new Point(Panel.LEFT_MARGIN, 20),
+                Parent = hPanel,
+            };
+            var musicianCategories = new Menu
+            {
+                Size = menuSection.ContentRegion.Size,
+                MenuItemHeight = 40,
+                Parent = menuSection
+            };
+            var lPanel = BuildLibraryPanel(wndw);
+            var library = musicianCategories.AddMenuItem("Library");
+            library.LeftMouseButtonReleased += delegate { wndw.Navigate(lPanel); };
+            var cPanel = BuildComposerPanel(wndw);
+            var composer = musicianCategories.AddMenuItem("Composer");
+            composer.LeftMouseButtonReleased += delegate { wndw.Navigate(cPanel); };
+
+            return hPanel;
+        }
+        private Panel BuildLibraryPanel(WindowBase wndw)
+        {
+            var lPanel = new Panel()
+            {
+                CanScroll = false,
+                Size = wndw.ContentRegion.Size
+            };
+            var backButton = new BackButton(wndw)
+            {
+                Text = "Musician",
+                NavTitle = "Library",
+                Parent = lPanel,
+                Location = new Point(20, 20),
+            };
+            var melodyPanel = new TintedPanel()
+            {
+                Location = new Point(0, Panel.BOTTOM_MARGIN + backButton.Bottom),
+                Size = new Point(lPanel.Width, lPanel.Size.Y - 50 - Panel.BOTTOM_MARGIN),
+                Parent = lPanel,
+                ShowBorder = true,
+                CanScroll = true
+            };
+
+            // Load local sheet music (*.xml) files.
+            Task loader = Task.Run(() => Sheets = xmlParser.LoadDirectory(GameService.FileSrv.BasePath + @"\musician"));
```

Comment:
If you end up loading panels during `LoadAsync`, you could probably turn this into its own async method or something and just `await` it.  You'd need to make the BuildPanel method async as well, but that shouldn't be an issue, I wouldn't imagine.